integration/service/oasis/stationgraph/station_graph.go

```diff
@@ -31,36 +31,33 @@ func NewStationGraph(currEnergyLevel, maxEnergyLevel float64, strategy chargings
 func (sg *stationGraph) setStartAndEndForGraph(currEnergyLevel, maxEnergyLevel float64) bool {
 	startLocation := sg.querier.GetLocation(stationfindertype.OrigLocationID)
 	if startLocation == nil {
-		glog.Errorf("Failed to find %#v from Querier GetLocation()\n", stationfindertype.OrigLocationID)
+		glog.Errorf("Failed to find %#v from Querier GetLocation()\n", stationfindertype.OrigLocationID.String())
```

Comment:
Use `%s` instead, then you don't need to explicitly call `String()`.

Reply:
Agree, remove .String()